app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,15 @@
+<section>
+    <div>
+        <h3>Post #<%=post.id %> </h3>
+        <p><%=post.Text%></p>
+        <div>
+            <p>Comments: <%=post.CommentsCounter %></p>
+            <p>Likes: <%=post.LikesCounter %></p>
+        </div>
```

Comment:
- [ ] [REQUIRED] Please make sure that likesCounter and CommentsCounter display the correct number. Please do it in each page that needs it